crates/cast/bin/cmd/wallet/mod.rs

```diff
@@ -151,6 +151,22 @@ pub enum WalletSubcommands {
     /// Derives private key from mnemonic
     #[command(name = "derive-private-key", visible_aliases = &["--derive-private-key"])]
     DerivePrivateKey { mnemonic: String, mnemonic_index: Option<u8> },
+
+    /// Decrypt a keystore file to get the private key
+    #[command(name = "decrypt-keystore", visible_alias = "dk")]
+    DecryptKeystore {
+        /// The name for the account in the keystore.
+        #[arg(value_name = "ACCOUNT_NAME")]
+        account_name: String,
+        /// If provided, keystore will be saved here instead of the default keystores directory
```

Comment:
I think, for this command, this comment is innacurate?

Reply:
You're definitely right, missed that. Fixed now.